### PR TITLE
Package name change

### PR DESCRIPTION
**Pull request by FranklinTrevor**
This pull request changes the TACOCAT/src package into LoadedTACO/src
-The package had to be renamed so TACOCAT can be imported for integration testing. Before the module and the package had the same name and since packages get import priority over modules TACOCAT can not be imported
-Every import was changed to LoadedTACO and the code was ran to confirm everything runs well
-Flux integration test script will be finished in a following pull request

### CODE REVIEW

File: TACOCAT.py [hunk 1]
```diff
@@ -26,6 +26,10 @@
 #----------------------------------------------------------------------------------#
 ## General Core Information
 
+def test(x,y):
```
Comment (lcarasik) [inline TACOCAT.py]: What is this test here?

File: TACOCAT_Read_In_File.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Reactor_Title = "Placeholder"
 ## Print Logicals for saving plots and data files 
 # (0 - save, 1 or higher - do not save)
-print_logic = 1
+print_logic = 0
```
Comment (lcarasik) [inline TACOCAT_Read_In_File.py]: Change this back to 1.

File: TACOCAT.py (hunk 1 above)
Comment (FranklinTrevor) [reply TACOCAT.py]: I added this when I was first resolving a importing issue and forgot to remove it. It should be removed now

File: TACOCAT_Read_In_File.py (hunk 2 above)
Comment (FranklinTrevor) [reply TACOCAT_Read_In_File.py]: Done

**Review by lcarasik — APPROVED** [review]
A simple name change now!

**Comment by lcarasik** [comment]
It's good to merge when ready @FranklinTrevor

**Pull request merged**